### PR TITLE
Support configuration of components' health probe delay

### PR DESCRIPTION
**Pull request by sophokles73**
The chart now allows configuration of the initialDelaySeconds property
of the readiness/liveness probes of the components.

The app version has been updated to the latest service release 1.5.1 as
well.

### CODE REVIEW

File: charts/hono/values.yaml [hunk 1]
```diff
@@ -157,6 +182,12 @@ dataGridExample:
   # imageName contains the name (including tag)
   # of the container image to use for the example data grid.
   imageName: infinispan/server:11.0.7.Final-1
+  # livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+  # configuration property of the component's liveness probe.
+  livenessProbeInitialDelaySeconds: 600
```
Comment (calohmn) [inline charts/hono/values.yaml:187]: Using 600s here for infinispan seems quite high, considering that the previous value was 50s.

File: charts/hono/templates/artemis/artemis-deployment.yaml [hunk 2]
```diff
@@ -53,14 +54,13 @@ spec:
           {{- . | toYaml | nindent 10 }}
         {{- end }}
         livenessProbe:
-          initialDelaySeconds: 40
+          initialDelaySeconds: {{ .Values.amqpMessagingNetworkExample.broker.artemis.livenessProbeInitialDelaySeconds }}
```
Comment (calohmn) [inline charts/hono/templates/artemis/artemis-deployment.yaml:57]: `.Values.amqpMessagingNetworkExample.broker.artemis.livenessProbeInitialDelaySecond` 
could be replaced by `$args.componentConfig.livenessProbeInitialDelaySecond`, making it shorter.

File: charts/hono/values.yaml [hunk 3]
```diff
@@ -68,6 +81,12 @@ amqpMessagingNetworkExample:
     # The default value ("Eclipse IoT;Hono") includes all adapters that authenticate with their
     # demo certificate.
     adapterUids: "Eclipse IoT;Hono"
+    # livenessProbeInitialDelaySeconds contains the value to use for the "initialDelaySeconds"
+    # configuration property of the component's liveness probe.
+    livenessProbeInitialDelaySeconds: 600
```
[the remark this answers is not in the document]
Comment (sophokles73) [reply charts/hono/values.yaml:86]: The reason for aiming that high is that when deploying the chart all pods are competing for the same (sparse) CPU  resources and therefore easily fail to respond to the liveness probe with an OK status. This leads to the component being restarted by kubernetes, which puts even more pressure on the CPUs. In addition, it is rather unlikely that the components will become unresponsive immediately during startup. FMPOV it therefore makes sense to start probing for liveness a lot after the ready state has been reached in order to provide for a more predictable start-up process.

**Comment by sophokles73** [comment]
@calohmn I have pushed changes ...

File: charts/hono/values.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (calohmn) [reply charts/hono/values.yaml:86]: Yes, I see. I still think that 10min for the (rather fast starting) dispatch router compared to the slower Artemis or also the spring-boot protocol adapters looks a bit strange though. But ok, I'm not religious about it.

**Review by calohmn — APPROVED** [review]
LGTM

File: charts/hono/values.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (sophokles73) [reply charts/hono/values.yaml:86]: They only start fast when there are lots of resources ...
How about setting it to 300s? Would that be ok for you?

[the remark this answers is not in the document]
Comment (calohmn) [reply charts/hono/values.yaml:86]: Yes, good.

**Pull request merged**